src/UI/class-settings-page.php

```diff
@@ -672,18 +847,84 @@ public function validate_options( array $input ): array {
 				);
 			}
 		}
-		// these can't be null, if somebody accidentally deselected them just reset to default.
-		if ( ! isset( $input['track_post_types'] ) ) {
-			$input['track_post_types'] = array( 'post' );
-		}
-		if ( ! isset( $input['track_page_types'] ) ) {
-			$input['track_page_types'] = array( 'page' );
+
+		$input['api_secret'] = sanitize_text_field( $input['api_secret'] );
+
+		if ( ! empty( $input['metadata_secret'] ) ) {
+			if ( strlen( $input['metadata_secret'] ) !== 10 ) {
+				add_settings_error(
+					Parsely::OPTIONS_KEY,
+					'metadata_secret',
+					__( 'Metadata secret is incorrect. Please contact Parse.ly support!', 'wp-parsely' )
+				);
+			} elseif ( isset( $input['parsely_wipe_metadata_cache'] ) && 'true' === $input['parsely_wipe_metadata_cache'] ) {
+				delete_post_meta_by_key( 'parsely_metadata_last_updated' );
+
+				wp_schedule_event( time() + 100, 'everytenminutes', 'parsely_bulk_metas_update' );
```

Comment:
Why 100 seconds? How is setting up a corn job related to adding Screen Options toggles?

Reply:
I see that this is just moved code, not new code.